acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/Signer.java

```diff
@@ -38,22 +40,27 @@
   private final EthSignerProcessRunner runner;
   private final Transactions transactions;
   private final Web3j jsonRpc;
+  private final HttpService web3jHttpService;
```

Comment:
looks like this doesn't need to be a field and can just be a local variable in the constructor

Reply:
Done.